skore/src/skore/sklearn/_plot/metrics/confusion_matrix.py

```diff
@@ -0,0 +1,251 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from sklearn.metrics import confusion_matrix as sklearn_confusion_matrix
+
+from skore.sklearn._plot.base import Display
+from skore.sklearn._plot.style import DEFAULT_STYLE
+
+
+class ConfusionMatrixDisplay(Display):
+    """Display for confusion matrix.
+
+    Parameters
+    ----------
+    confusion_matrix : ndarray of shape (n_classes, n_classes)
+        Confusion matrix.
+
+    display_labels : list of str, default=None
+        Display labels for plot. If None, display labels are set from 0 to
+        ``n_classes - 1``.
+
+    include_values : bool, default=True
+        Includes values in confusion matrix.
+
+    cmap : str or matplotlib Colormap, default='viridis'
+        Colormap used for confusion matrix. If None, 'viridis' is used.
+
+    normalize : {'true', 'pred', 'all'}, default=None
+        Normalizes confusion matrix over the true (rows), predicted (columns)
+        conditions or all the population. If None, confusion matrix will not be
+        normalized.
+
+    values_format : str, default=None
+        Format specification for values in confusion matrix. If None, the format
+        specification is 'd' or '.2g' whichever is shorter.
+
+    ax : matplotlib axes, default=None
+        Axes object to plot on. If None, a new figure and axes is created.
+
+    colorbar : bool, default=True
+        Whether or not to add a colorbar to the plot.
+
+    Attributes
+    ----------
+    figure_ : matplotlib Figure
+        Figure containing the confusion matrix.
+
+    ax_ : matplotlib Axes
+        Axes with confusion matrix.
+
+    text_ : ndarray of shape (n_classes, n_classes), dtype=matplotlib Text or \
+            None
+        Array of matplotlib text elements containing the values in the
+        confusion matrix.
+
+    """
+
+    def __init__(
```

Comment:
```suggestion
    @StyleDisplayMixin.style_plot
    def __init__(
```
This makes the style of all skore plots consistent.